packages/permission-controller/src/PermissionController.ts

```diff
@@ -487,12 +492,13 @@ type MergeCaveatResult<CaveatType extends CaveatConstraint | undefined> =
 export type ExtractPermission<
   ControllerPermissionSpecification extends PermissionSpecificationConstraint,
   ControllerCaveatSpecification extends CaveatSpecificationConstraint,
-> = ControllerPermissionSpecification extends ValidPermissionSpecification<ControllerPermissionSpecification>
```

Comment:
These were fixed by Prettier automatically. it seemed easiest to leave it in, despite it being unrelated to the other changes.